### PR TITLE
Update react-native-libraries.json

### PR DESCRIPTION
**Pull request by aravind3566**
# 📝 Why & how

# ✅ Checklist

- [ ] Added library to **`react-native-libraries.json`**
- [ ] Updated library in **`react-native-libraries.json`**

- [ ] Documented in this PR how to use the feature or replicate the bug.
- [ ] Documented in this PR how you fixed or created the feature.

### CODE REVIEW

File: react-native-libraries.json [hunk 1]
```diff
@@ -13188,6 +13188,10 @@
   {
     "githubUrl": "https://github.com/aravind3566/react-native-in-app-updates",
     "npmPkg": "react-native-in-app-updates",
+    "images": [
+       "https://camo.githubusercontent.com/451499c72e2d84829bf93167d97af95c1ce6b0234baa3aa07599bb5cfe6818d7/68747470733a2f2f646576656c6f7065722e616e64726f69642e636f6d2f7374617469632f696d616765732f6170702d62756e646c652f666c657869626c655f666c6f772e706e67",
+       "https://camo.githubusercontent.com/18561ffcd55665a885352bb929c3b08a50ee8858918182e1e382832e78eec703/68747470733a2f2f646576656c6f7065722e616e64726f69642e636f6d2f7374617469632f696d616765732f6170702d62756e646c652f696d6d6564696174655f666c6f772e706e67"
+     ],
```
Comment (Simek) [inline react-native-libraries.json:13194]: The CI fails because GitHub Camo URLs won't work permanently.

Comment (aravind3566) [reply react-native-libraries.json:13194]: @Simek no issue.,

File: react-native-libraries.json [hunk 2]
```diff
@@ -13188,6 +13188,10 @@
   {
     "githubUrl": "https://github.com/aravind3566/react-native-in-app-updates",
     "npmPkg": "react-native-in-app-updates",
+    "images": [
+       "https://camo.githubusercontent.com/451499c72e2d84829bf93167d97af95c1ce6b0234baa3aa07599bb5cfe6818d7/68747470733a2f2f646576656c6f7065722e616e64726f69642e636f6d2f7374617469632f696d616765732f6170702d62756e646c652f666c657869626c655f666c6f772e706e67",
+       "https://camo.githubusercontent.com/18561ffcd55665a885352bb929c3b08a50ee8858918182e1e382832e78eec703/68747470733a2f2f646576656c6f7065722e616e64726f69642e636f6d2f7374617469632f696d616765732f6170702d62756e646c652f696d6d6564696174655f666c6f772e706e67"
```
Comment (Simek) [inline react-native-libraries.json:13193]:
```suggestion
       "https://developer.android.com/static/images/app-bundle/flexible_flow.png",
       "https://developer.android.com/static/images/app-bundle/immediate_flow.png"
```

Looks like we can use the origin URL, before anonymization.

Comment (aravind3566) [reply react-native-libraries.json]: @Simek thank you.,

**Review by Simek — APPROVED** [review]
Thanks for the updates, LGTM! 👍

**Pull request merged**